dialogues/target_detection_dialog.py

```diff
@@ -158,38 +229,9 @@ def set_threshold_by_input(self):
             
     def get_threshold(self):
         return self.threshold
-    
-    def generate_result_images(self):
-        print(self.get_timestamp() + " Generating...")
-        
-        self.corr_raw = self.correlation_coefficients()
-        self.corr_img = self.raw_to_img(self.corr_raw, cv2.COLORMAP_JET)
-     
-        self.corr_thresh_raw = self.threshold_correlation_coefficients()
-        self.corr_thresh_img = self.raw_to_img(self.corr_thresh_raw, cv2.COLORMAP_BONE)
-        
-        height, width, _ = self.corr_img.shape
-        bytes_per_line = 3 * width
-        self.corr_q_img = QImage(self.corr_img.data, width, height, bytes_per_line, QImage.Format_RGB888).rgbSwapped()
-        self.corr_thresh_q_img = QImage(self.corr_thresh_img.data, width, height, bytes_per_line, QImage.Format_RGB888).rgbSwapped()
-        
-        self.update_result_image()
-        self.generated = True
-    
-    def raw_to_img(self, raw, colormap):
-        img = cv2.normalize(raw, None, 0, 255, cv2.NORM_MINMAX, cv2.CV_8UC1)
-        img = cv2.applyColorMap(img, colormap)
-        
-        return img
-        
-    def update_result_image(self):
-        if self.rb_thresholded.isChecked():
-            self.lbl_result_image.setPixmap(QPixmap.fromImage(self.corr_thresh_q_img))
-        else:
-            self.lbl_result_image.setPixmap(QPixmap.fromImage(self.corr_q_img))
 
-    def show_original_image(self):
-        if self.path_to_file == "":
+    def show_original_image(self, show_dot = True):
```

Comment:
img is used instead of image through most of your code, keep one wording

Reply:
Fixed